Bool player_node_rdy